docs/network.md

```diff
@@ -0,0 +1,77 @@
+# 7. Adding new edges to routing tables
+
+This section covers the process of adding new edges, received from another nodes,
+to the routing table. It consists of several steps covered below.
+
+## 7.1 Step 1
+`PeerManagerActor` receives `RoutingTableSync` message containing list of new `edges` to add.
+This message is them forwarded to `RoutingTableActor`.
+Field `accounts` will be covered in a different section.
```

Comment:
Could we add the section number?

Reply:
@bowenwang1996 What do you mean?

Reply:
Ok, I added TODO, I'm not sure in which section that will be. My plan is to ship documentation in parts. And then go back to it again, and make fixes, add missing sections, at TODOs. etc.